worker/pkg/workflows/datasync/workflow/workflow.go

```diff
@@ -319,6 +325,30 @@ func retrieveActivityOptions(
 	return actOptResp, nil
 }
 
+func runPostTableSyncActivity(
+	ctx workflow.Context,
+	logger log.Logger,
+	actOptResp *syncactivityopts_activity.RetrieveActivityOptionsResponse,
+	jobId string,
+	name string,
+) error {
+	logger.Debug("executing post table sync activity")
+	var resp *posttablesync_activity.RunPostTableSyncResponse
+	var postTableSyncActivity *posttablesync_activity.Activity
+	err := workflow.ExecuteActivity(
+		workflow.WithActivityOptions(ctx, *actOptResp.SyncActivityOptions),
```

Comment:
Is it expected here that we want to use the SyncActivityOptions for this activity?